product_docs/docs/epas/14/epas_compat_reference/02_the_sql_language/03_functions_and_operators/10_conditional_expressions.mdx

```diff
@@ -129,13 +129,24 @@ The `NVL` function returns the first of its arguments that is not null. `NVL` ev
 NVL(expr1, expr2)
 ```
 
-The return type is the same as the argument types; all arguments must have the same data type (or be coercible to a common type). `NVL` returns `NULL` if all arguments are `NULL`.
+The return type is the same as the argument types; all arguments must have the same data type (or be coercible to a common type). `NVL` returns `NULL` if all arguments are `NULL`. `''` is considered as unknown and if they are not coercible to the common data type then it throws an error.
```

Comment:
What does "they" refer to in the last sentence? Without being explicit it is a little hard to parse the sentence. Is it "all arguments?"

Reply:
changed and committed